packages/clay-form/CHECKBOX.mdx

```diff
@@ -32,22 +41,36 @@ Use [`checked`](#api-checked) property to check or not the checkbox.
 
 <Checkbox />
 
+## Advanced Usages
+
+Here you can see some options or ideas on how to extend our Checkbox component to fit your use.
+
+### Indeterminate Checkbox
+
 Use [`indeterminate`](#api-indeterminate) property for making the checkbox indeterminate.
 
 <CheckboxIndeterminate />
 
+### Checkbox with a callback
+
 To set a callback function when the value of the checkbox is changed, use [`onChange`](#api-onChange) property. See the following example:
 
 <CheckboxWithCallback />
 
+### Checkbox with a custom container
+
 If you want to modify the container that wraps the checkbox, use [`containerProps`](#api-containerProps):
 
 <CheckboxContainerProps />
 
+### Checkbox with a label
+
 Use [`label`](#api-label) property to describe what the checkbox is for.
 
 <CheckboxLabel />
 
+### Inline Checkboxes
```

Comment:
For all of these titles, I think adding in "Checkbox" is redundant since they are on the checkbox page. They could simply be `Container Props... Label... Inline...`